### ISSUE TITLE
No shell script in release

### ISSUE THREAD

**Issue by jpl-jengelke** [digital-preservation/droid#802]
There is no `droid.sh` script in the unpacked release ZIP as [the instructions state](https://github.com/digital-preservation/droid#linux--osx--windows-users). Is it needed to properly run the application? I was able to launch it directly on OS X by executing the GUI JAR. Please fix. Thanks.

**Comment by jpl-jengelke** [comment]
Related: https://github.com/digital-preservation/droid/issues/636

**Comment by Dclipsham** [comment]
Hi John, where did you download DROID from? 

On the download page (https://www.nationalarchives.gov.uk/information-management/manage-information/preserving-digital-records/droid/) there are two versions available:

The 'Windows-only' version does not contain the droid.sh file (use the droid.bat file to launch DROID in this case).
The 'all platforms' version, does contain droid.sh as per the attached image. 
![image](https://user-images.githubusercontent.com/2189778/179574468-b79505d5-635e-45cc-9404-4c987565a93d.png)

I hope this is useful.
David

**Comment by jpl-jengelke** [comment]
Thanks. I downloaded the first asset from the release bundle at https://github.com/digital-preservation/droid/releases/tag/droid-6.5 . I see now that this Java application is built with a Windows and a non-windows release distribution. I'd imagine it might be easier to package the start scripts in one bundle but perhaps there are other differences in the distribution. 

Thanks for clarifying. Closing this.
